docroot/themes/humsci/humsci_basic/templates/contrib/colorbox-formatter.html.twig

```diff
@@ -0,0 +1,26 @@
+{#
+/**
+ * @file
+ * Default theme implementation to display a formatted colorbox image field.
+ *
+ * Available variables:
+ * - image: A collection of image data.
+ * - url: An URL the image can be linked to.
+ * - attributes: Link attributes.
+ *
+ * @see template_preprocess_colorbox_formatter()
+ *
+ * @ingroup themeable
+ */
+#}
+
+{# set a generic link label in case there is no alt attribute on the image or a caption #}
+{% set link_label = "Open image in the gallery" %}
+{# If the image has an alt attribute, use it as the link label. #}
+{% if attributes['data-cbox-img-attrs']|render != '{&quot;alt&quot;:&quot;&quot;}' %}
+  {% set link_label = "Open " ~ attributes['data-cbox-img-attrs'] ~ " image in the gallery" %}
+{% endif %}
+{% set link_label = link_label|raw %}
+{% set link_label = link_label|replace({'{&quot;alt&quot;:&quot;':'', '&quot;}':''}) %}
```

Comment:
This works, but checking the colorbox module code I found that `data-cbox-img-attrs` can include more attributes, not only `alt`. To prevent weird values, you can get the alt text from the `image` variable:

```
{% set alt_text = image['#alt'] %}
```

That would also make the use of the `|raw` filter unnecessary.

Reply:
Oh my, you are right, we don't need that stuff anymore. Thanks!